plugins/seccomp/src/PoliciesTable.tsx

```diff
@@ -0,0 +1,372 @@
+            No
+          </Button>
+        </>
+      ),
+      persist: true,
+    });
+  }
+
+  function policyDelete(
+    policiesToDelete: any,
+    selectedPolicies: string[],
+    resetSelectedPolicies: () => void
+  ) {
+    const key = enqueueSnackbar('Delete in progress', { variant: 'warning', persist: true });
+    const deletePolicyApi = policiesToDelete.map((policy: any) => {
+      return deleteSeccompPolicy(policy);
+    });
+
+    Promise.all(deletePolicyApi)
+      .catch(error => {
+        console.error(error);
+        return deletePolicyApi;
+      })
+      .then(() => {
+        closeSnackbar(key);
+        let tempSelectedPolicies = [...selectedPolicies];
+        resetSelectedPolicies();
+        enqueueSnackbar(
+          `Deleted Policies ${policiesToDelete
+            .map((policy: any) => policy?.metadata?.name)
+            .join(', ')} successfully`,
+          {
+            variant: 'success',
+          }
+        );
+        // refetch policy list to get the updated set of policies
+        setMassagedSeccompPolicies(massagedSeccompPolicies =>
+          massagedSeccompPolicies.filter(
+            (policy: any) => !tempSelectedPolicies.includes(policy?.metadata?.uid)
+          )
+        );
+        setMassagedAppliedPolicies(massagedAppliedPolicies =>
+          massagedAppliedPolicies.filter(
+            (policy: any) => !tempSelectedPolicies.includes(policy?.metadata?.uid)
+          )
+        );
+      });
+  }
+
+  return (
+    <Box py={1}>
+      <PolicyTableViewGenerator
+        title="Seccomp Policies"
+        tableData={massagedSeccompPolicies}
+        setIsEditorOpen={setIsEditorOpen}
+        setEditorValue={setEditorValue}
+        handlePolicyDelete={handlePolicyDelete}
+        shouldHaveApply={true}
+        callback={(item: any) =>
+          setMassagedSeccompPolicies(
+            massagedSeccompPolicies.filter(
+              (policyItem: any) => policyItem.metadata.name !== item?.metadata.name
+            )
+          )
+        }
+      />
+      <PolicyTableViewGenerator
+        title="Applied Policies List"
+        tableData={massagedAppliedPolicies}
+        setIsEditorOpen={setIsEditorOpen}
+        setEditorValue={setEditorValue}
+        handlePolicyDelete={handlePolicyDelete}
+      />
+      <PolicyTableViewGenerator
+        title={
+          <SectionHeader
+            title={
+              <>
+                Dangling Policies
+                <Tooltip title="Policies for which a Pod couldn't be found">
+                  <IconButton>
+                    <Icon icon={infoIcon} width="20" height="20" />
+                  </IconButton>
+                </Tooltip>
+              </>
+            }
+          />
+        }
+        tableData={danglingPolicies}
+        setIsEditorOpen={setIsEditorOpen}
+        setEditorValue={setEditorValue}
+        handlePolicyDelete={handlePolicyDelete}
+      />
+      {editorValue && (
+        <EditorDialog
```

Comment:
I couldn't see the notification with the grace period and further success when successfully changing the policy with the dialog.